web/src/main/java/com/graphhopper/http/DefaultModule.java

```diff
@@ -52,6 +52,9 @@ public GraphHopper getGraphHopper() {
      */
     protected GraphHopper createGraphHopper(CmdArgs args) {
         GraphHopper tmp = new GraphHopperOSM().forServer().init(args);
+        //TODO We should set the BBox and resolution
+        //TODO, move to a more appropriate place
+        tmp.setSpatialRuleLookup(SpatialRuleLookupBuilder.build());
```

Comment:
related to the TODOs :) ... it is suboptimal that it cannot be configured where to load the json from and the resolution ...